### PR TITLE
Update routing at Candlestick 

### PR DESCRIPTION
**Pull request by BobSiegel**

### CODE REVIEW

File: cue-sheets/hike-north.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Walk Northbound
 
-Crosstown Trail Walk Cue Sheet, Northbound
+Crosstown Trail Walk Cue Sheet, Sorthbound
```
Comment (matthewblain) [inline cue-sheets/hike-north.md]: You should fix the spellliiinng here!

**Review by matthewblain — CHANGES_REQUESTED** [review]
try again!

**Review by matthewblain — COMMENTED** [review]
try now

File: cue-sheets/hike-south.md [hunk 2]
```diff
@@ -176,8 +176,11 @@ Glen Park BART to Sunrise Point/Candlestick Point State Recreation Area
...
-* At white gates, turn right to continue on Executive Park Blvd. Descend one block, and cross Harney Way to enter Candlestick Point State Recreation Area.
-* Turn left on wide dirt path and walk past a parking lot that will soon be on your left.
-* At fork, bear left on narrow dirt trail, and shortly thereafter, follow path on right side of white traffic barriers.
-* Meander for about half a mile along dirt and asphalt trails closest to the shore of SF Bay.
-* Section ends at paved roundabout at Sunrise Point. (**Restrooms**, picnic tables, fishing pier, campground by reservation.)
+* At white gate, turn right to continue on Executive Park. Descend one block, and cross Harney to enter Candlestick Point State Recreation Area.
+* Turn R at Crosstown Trail signpost.
+* At next Crosstown Trail signpost located at beginning of colored concrete, turn L to follow dirt path going east with the bay on your right.
+* At Y, bear R onto narrow path that proceeds toward white traffic barriers.
+* Turn R at Crosstown Trail signpost to follow dirt path to R of white traffic barriers.
```
Comment (matthewblain) [inline cue-sheets/hike-south.md:183]: is this really a turn? or 'continue to R'?

**Review by matthewblain — APPROVED** [review]
Looks good. Would you like to share it with others first, they can see the diffs too?

**Comment by BobSiegel** [comment]
No need to share it with others.  Upload with fervor!

**Pull request merged**